### DISCUSSION TITLE
Is anyone using web3wallet and web3modal+wagmi in same RN application

### DISCUSSION THREAD

**Discussion by shan-c** [WalletConnect/walletconnect-monorepo#4196] [Developer Support] [answered]
I'm trying to integrate web3Modal+wagmi into my existing react native application that already uses web3wallet. Web3Modal part of the code works fine but i observe a lot of errors like 'No matching key' and 'this.events.off is not a function'  in web3Wallet and iam unable to connect to any dApp now. It was all working fine before integrating web3Modal + wagmi

**Comment by glitch-txs** [comment, marked as answer]
they share the same store so it's not possible at them moment to integrate them both simultaneously. You can request support for this by opening an issue in the [monorepo](https://github.com/WalletConnect/walletconnect-monorepo)
